### ISSUE TITLE
What is "Some really important Bugs are now fixed" of Auto Dark Mode 2.1.1 ?

### ISSUE THREAD

**Issue by matsuda0707** [Armin2208/Windows-Auto-Night-Mode#21]
Yesterday I installed Auto Dark Mode 2.1 and it seemed working OK. 
But when I restarted the system around 22:00, the system could not activate a crtain application wgih I have been using for about 20 years (very old, but stable application). After some cut and try, I found that Auto Dark Mode 2.1 may be related (this is only one program I installed yesterday).
So I uninstalled Auto Dark Mode 2.1 and checked my system by restating and everything worked OK.
Then I found new Auto Dark Mode 2.1.1 is released and installed it, but no problem was recreated and worked OK.
The affeted my application uses VB6 runtime library SP5.
So could you explain what is "Some really important Bugs are now fixed" and affection of this bug?

**Comment by Armin2208** [comment]
In 2.1.1 I fixed a Issue with the Background Location Updater. It puts the wrong Times to the Tasks, and it activates every week. So people using 2.1.0 with location service would get a broken theme switch every week.

I also fixed some issues with the selection of System/Apps/Edge for Always white, Always dark and Fit to theme. Plus I made several improvements to the Accent Color-Feature.

Last I fixed a issue that's there since 1.0. While activating the background task, the App also runned some unnessecary code, because it was in the main. I honestly never noticed that. Now this is fixed and for me, this is a really really important fix. Theme Switch is now much more efficiency, because is only runs what's really needed.

I don't see any relation to other software honestly. Maybe it was some bad luck. 

If you still experience any issue (with your app) let me know.
